### PR TITLE
taking Gestures View

### PR DESCRIPTION
**Pull request by alighayeni**

### CODE REVIEW

**Comment by alighayeni** [comment]
Hi @barbaramartina ,  
I hope you are doing well. 
In the GestureView class, you put two links,
https://developer.apple.com/documentation/swiftui/adding-interactivity-with-gestures (I did add all the samples for this link) 

but for the second link, I would like to add another subcategory in the Gesture section and add this part in a separate view
https://developer.apple.com/documentation/swiftui/composing-swiftui-gestures 
do you think it is a good idea or I should add that in the view which I already created?

**Comment by alighayeni** [comment]
I already create another section and PR for composing gestures view. please check and feel free to modify that if you think it needed some changes.

**Comment by barbaramartina** [comment]
@alighayeni all right here, at the moment traveling. Thanks for asking, yes it's great idea, some topics have so much to show. I did something similar with the animation when I realised adding all together in one view will be too much.

File: SwiftUICatalog/Views/Sections/Gestures/GesturesView.swift [hunk 1]
```diff
@@ -37,8 +37,159 @@ import SwiftUI
...
+            }
+            /**
+             - Update Permanent State During a Gesture
+             To track changes to a gesture that shouldn’t reset once the gesture ends, use the onChanged(_:) callback. For example, to count the number of times your app recognizes a long press, add an onChanged(_:) callback and increment a counter.
+             */
+            .onChanged {_ in
+                self.totalNumberOfTaps += 1
+            }
+            /**
+             - Update Permanent State When a Gesture Ends
```
Comment (barbaramartina) [inline SwiftUICatalog/Views/Sections/Gestures/GesturesView.swift:70]: ah nice to include some documentation comments block, will take the idea for other future additions I do.

**Pull request merged**